input/fsh/aliases.fsh

```diff
@@ -0,0 +1,3 @@
+Alias: $v3-ParticipationType = http://terminology.hl7.org/CodeSystem/v3-ParticipationType
+Alias: $NotificationInfoExtension = http://example.org/fhir/StructureDefinition/NotificationInfoExtension
+Alias: $classification-view-page.xhtml = https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=1943&versionKey=2203
```

Comment:
I'd name this alias differently. `$codeserver-booking-status-page` or something similar.

OK to pull this in, as this is mainly importing the gofsh outputs, but making a note here so we'll hopefully rememeber to change it afterwards.